cluster/manifests/daemonupgrader/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: daemonupgrader
+  namespace: kube-system
+  labels:
+    application: daemonupgrader
+    version: 8264ca0
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: daemonupgrader
+  template:
+    metadata:
+      labels:
+        application: daemonupgrader
+        version: 8264ca0
+    spec:
+      containers:
+      - name: daemonupgrader
+        image: pierone.stups.zalan.do/teapot/k8s-daemonupgradecontroller:8264ca0
```

Comment:
The image should be pulled from the opensource registry, otherwise it will fail to launch when setting up a new cluster.